### DISCUSSION TITLE
Lien entre Mes Achats et le diag?

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1233] [Question / réponse]
Question d'une utilisatrice (L.A.) : les chiffres du diagnostic sont-ils automatiquement renseignés à partir des montants de factures renseignés dans Mes Achats ?

**Comment by alemangui** [comment]
Hello, oui c'est bien ça. Si un utilisateur a des achats pour une année et cantine, lors de la création du diagnostic pour cette année et cantine des aides de remplissage sont affichées. Par exemple :

![achats-diag](https://user-images.githubusercontent.com/1225929/160644244-15ffe453-98df-4d11-947d-6bf76351cbb5.gif)

**Reply by alemangui** [reply]
@valeriemerle on parlait de cette fonctionnalité à Dormans, voici un gif :wink:

**Reply by ghost** [reply]
@alemangui, merci pour le gif! C'est limpide.
